src/libraries/System.Runtime/tests/System/ArrayTests.cs

```diff
@@ -2945,7 +2945,8 @@ public static void IStructuralComparable_Invalid()
         [Fact]
         public static void IStructuralComparable_NullComparer_ThrowsNullReferenceException()
         {
-            // This was not fixed in order to be compatible with the full .NET framework and Xamarin. See #13410
+            // This was not fixed in order to be compatible with the full .NET framework and Xamarin.
```

Comment:
```suggestion
            // This was not fixed in order to be compatible with the .NET Framework and Xamarin.
```
I have been updating .NET Framework references to use the proper official name as I run into them.

Reply:
I updated a bunch.